### PR TITLE
update jetson agx benchmark

### PR DESCRIPTION
**Pull request by zineos**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=1963) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=1963) before we can accept your contribution.<br/><hr/>**neos** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=1963) it.</sub>

File: benchmark/README.md [hunk 1]
```diff
@@ -1204,3 +1204,159 @@ cooling_down = 0
...
+          vgg16_int8  min =  872.47  max =  886.24  avg =  880.69
+            resnet50  min =   13.92  max =   40.74  avg =   30.52
+       resnet50_int8  min =  466.06  max =  476.91  avg =  470.40
+      squeezenet_ssd  min =   11.53  max =   38.27  avg =   18.68
+ squeezenet_ssd_int8  min =  129.05  max =  135.18  avg =  131.57
+       mobilenet_ssd  min =    7.69  max =   34.02  avg =   19.65
+  mobilenet_ssd_int8  min =  214.68  max =  219.59  avg =  217.03
+      mobilenet_yolo  min =   11.40  max =   39.92  avg =   24.83
+  mobilenetv2_yolov3  min =    7.70  max =   25.49  avg =   15.46
+         yolov4-tiny  min =   11.28  max =   34.75  avg =   22.27
```
Comment (nihui) [inline benchmark/README.md:1321]: The results here is very unstable, is there another process using the gpu? please retest and confirm it.

Comment (zineos) [reply benchmark/README.md:1321]: Maybe another process using the gpuI. I will retest and confirm it.

Comment (zineos) [reply benchmark/README.md]: The new test result is updated

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**